specfile/exceptions.py

```diff
@@ -23,3 +23,10 @@ def __str__(self) -> str:
             if line.startswith("error:"):
                 return line.split("error: ")[1]
         return "\n" + b"".join(self.stderr).decode()
+
+
+class RPMMacroRemovalException(RPMException):
+    """Exception related to failed removal of RPM macros."""
+
+    def __init__(self) -> None:
+        super().__init__([])
```

Comment:
Wouldn't it be better to either inherit from more generic exception or to modify `RPMException` so that the `stderr` attribute is optional? This would traceback with a newline, which could be a bit confusing, and I don't like it in general.
How about just `MacroRemovalException` inheriting from `SpecfileException`?